examples/context/theme-detailed-app.js

```diff
@@ -1,11 +1,11 @@
 import {ThemeContext, themes} from './theme-context';
 import ThemedButton from './themed-button';
 
-// An intermediate component that uses the ThemedButton
+// Un componente intermedio que utiliza ThemedButton.
 function Toolbar(props) {
   return (
     <ThemedButton onClick={props.changeTheme}>
-      Change Theme
+      Cambiar tema
```

Comment:
Solo estamos traduciendo los comentarios 🥇 

```suggestion
      Change Theme
```